spring-integration-core/src/main/java/org/springframework/integration/support/management/observation/IntegrationObservations.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.support.management.observation;
+
+import org.springframework.messaging.Message;
+
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+
+/**
+ * A convenient factory for {@link IntegrationObservation} instances.
+ *
+ * @author Artem Bilan
+ *
+ * @since 6.0
+ */
+public final class IntegrationObservations {
```

Comment:
As you see I had to create this factory for better target usage.
I really don't see reason in the `DefaultMessageReceiverObservationConvention` instance per request.
And I don't feel yet that I need some `customConvention`.

It probably would be great if `getDefaultConvention()` would return an instance instead of `Class` and this one would always be used in case of plain `observation(registry, context)` call.

@marcingrzejszczak , you mentioned in [Spring AMQP PR](https://github.com/spring-projects/spring-amqp/pull/1500#discussion_r965296869) that we always have to use a convention for contextual name propagation, but as you see it is a bit involved in the target code since we have to call the `observation()` method with much more arguments, than just rely on a default convention by default...

Reply:
>  I really don't see reason in the DefaultMessageReceiverObservationConvention instance per request.

We typically create singleton fields for those in instrumentations.

>  And I don't feel yet that I need some customConvention.

You don't, but the users might ;) What if they want to follow their internal naming convention and you don't give them such an option?

> It probably would be great if getDefaultConvention() would return an instance instead of Class and this one would always be used in case of plain observation(registry, context) call.

That will be harder to parse for the micrometer docs generator. 

> @marcingrzejszczak , you mentioned in https://github.com/spring-projects/spring-amqp/pull/1500#discussion_r965296869 that we always have to use a convention for contextual name propagation, but as you see it is a bit involved in the target code since we have to call the observation() method with much more arguments, than just rely on a default convention by default...

Yes, I mentioned that it would be preferable to use the `ObservationConvention` to name observations either via contextual or non-contextual methods. 

The way you did it ATM doesn't allow to set any customizations by the users at the level of Spring Integration. Those can be still done by `GlobalObservationConvention` and the `ObservationRegistry` (which may be OK if you're OK with that). That means that if I want to change the tags or naming for Spring Integration then I can ONLY do it by registering a proper `GlobalObservationConvention` and attaching it to the `ObservationRegistry`. The other option would be to give the user a setter to provide a custom implementation of your `MessageRecevierObservationConvention`. It's up to you to decide (e.g. Reactor Netty does not allow any customizations).